static/app/views/alerts/rules/metric/types.tsx

```diff
@@ -185,6 +185,17 @@ export const TargetLabel = {
   [TargetType.TEAM]: t('Team'),
 };
 
+export const PriorityOptions = {
+  [ActionType.PAGERDUTY]: ['critical', 'warning', 'error', 'info'],
+  [ActionType.OPSGENIE]: ['P1', 'P2', 'P3', 'P4', 'P5'],
+};
+
+// default priorities per threshold (0 = critical, 1 = warning)
+export const DefaultPriorities = {
+  [ActionType.PAGERDUTY]: {[0]: 'critical', [1]: 'warning'},
+  [ActionType.OPSGENIE]: {[0]: 'P1', [1]: 'P2'},
```

Comment:
I'd recommend adding some typing to this, then changing the data structure from an object to an array.

```suggestion
export const DefaultPriorities: Record<ActionType, string[]> = {
  [ActionType.PAGERDUTY]: ['critical', 'warning'],
  [ActionType.OPSGENIE]: ['P1', 'P2'],
```